velox/functions/prestosql/UuidFunctions.h

```diff
@@ -20,10 +20,46 @@
 
 #include "velox/functions/Macros.h"
 #include "velox/functions/Registerer.h"
+#include "velox/functions/prestosql/Comparisons.h"
 #include "velox/functions/prestosql/types/UuidType.h"
 
 namespace facebook::velox::functions {
 
+struct UuidCompareBase {
+  FOLLY_ALWAYS_INLINE int compare64(uint64_t x, uint64_t y)
+  {
+    return x < y ? -1 : (x == y ? 0 : 1);
+  }
+
+  int compare128(const int128_t& lhs, const int128_t& rhs) {
+    int comp = compare64(HugeInt::upper(lhs), HugeInt::upper(rhs));
+    if (comp != 0) {
+      return comp;
+    }
+    return compare64(HugeInt::lower(lhs), HugeInt::lower(rhs));
+  }
+};
+
+#define VELOX_GEN_BINARY_EXPR_UUID(Name, uuidCompExpr, TResult)               \
```

Comment:
TResult parameter is not used.

Reply:
Thanks, good catch! It's actually the same in `VELOX_GEN_BINARY_EXPR_TIMESTAMP_WITH_TIME_ZONE` too

Reply:
done